DECISIONS.md

```diff
@@ -1,4 +1,4 @@
 This is the file containing our decision and argumentations on those during our minitwit project
 
-REWRITING MINITWIT IN GO
+## REWRITING MINITWIT IN GO
 Go is really fast and scalable. It is a lightweight language. Many ways to use Google API's are often written i GO by some Google dev, it would be nice to be able to follow those implementations in the future. There is a hype around Go, and we want to see what the hype is about.
```

Comment:
``` suggestion
Go is really fast and scalable. It is a lightweight language. Many ways to use Google API's are often written in GO by some Google dev, it would be nice to be able to follow those implementations in the future. There is a hype around Go, and we want to see what the hype is about.
```